control/autonomous_emergency_braking/src/node.cpp

```diff
@@ -304,6 +318,8 @@ void AEB::onCheckCollision(DiagnosticStatusWrapper & stat)
 
 bool AEB::checkCollision(MarkerArray & debug_markers)
 {
+  using colorTuple = std::tuple<double, double, double, double>;
```

Comment:
FYI

A useful utility function for creating debug markers is available [here](https://github.com/autowarefoundation/autoware.universe/blob/a6c69122541b141cbbf3c37c03151a82009d50fa/control/vehicle_cmd_gate/src/marker_helper.hpp#L61).